### ISSUE TITLE
Localization missing in 0.17.0

### ISSUE THREAD

**Issue by pascalberger** [chocolatey/ChocolateyGUI#778]
Localizations seem to be missing with 0.17.0 installer.

While installing 0.16.0 localizations are correctly included (e.g. `\de\ChocolateyGui.resources.dll`). Installing 0.17.0 only comes with `Microsoft.VisualStudio.Threading.resources.dll` and `Microsoft.VisualStudio.Validation.resources.dll`, but misses `ChocolateyGui.resources.dll`. This seems to affect all languages.

**Comment by gep13** [comment]
@AdmiringWorm no, it wasn't anything to do with Wix.  The builds from AppVeyor will always contain the localized files, but the version pushed version of Chocolatey GUI doesn't come from AppVeyor, it comes from another machine, that hadn't exported from Transifex, and therefore hadn't been included in the released artifacts.  I am taking care of that today, and will be pushing a new version.

**Comment by AdmiringWorm** [comment]
Ah, I see. That makes sense of why they weren't included.
